mep/books/admin.py

```diff
@@ -455,41 +447,29 @@ def before_import_row(self, row, **kwargs):
         )
 
 
-def unique_list(l):
-    l2 = []
-    for x in l:
-        if x not in set(l2):
-            l2.append(x)
-    return l2
```

Comment:
Let's just take this out for now and we can use any duplicated output to spot any issues?

Reply:
I was thinking that too - it would actually be useful for a data editor to see the dupes in the output. 💡 🧠 👍